### PR TITLE
PWN-7571 - update supported tokens and use them in receive + add as a first tokens in receive

### PR DESCRIPTION
**Pull request by kabramovp2p**
## Jira Ticket

https://p2pvalidator.atlassian.net/browse/PWN-7571

## Description of Work

PWN-7571 - update supported tokens and use them in receive + add as a first tokens in receive

### CODE REVIEW

File: app/src/main/java/org/p2p/wallet/receive/tokenselect/ReceiveTokensPresenter.kt [hunk 1]
```diff
@@ -36,10 +37,14 @@ class ReceiveTokensPresenter(
     private var searchJob: Job? = null
 
     private val tokensFlow = MutableStateFlow<List<AnyCellItem>>(emptyList())
+    private lateinit var pinedWormholeTokens: List<TokenData>
+    private lateinit var pinedWormholeTokensAddresses: List<String>
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/receive/tokenselect/ReceiveTokensPresenter.kt:41]: `pinned`

File: app/src/main/java/org/p2p/wallet/receive/tokenselect/ReceiveTokensPresenter.kt [hunk 2]
```diff
@@ -57,6 +62,24 @@ class ReceiveTokensPresenter(
...
+                    mintAddress = token.mintAddress,
+                    name = erc20Token.replaceTokenName ?: token.tokenName,
+                    symbol = erc20Token.replaceTokenSymbol ?: token.tokenSymbol,
+                    iconUrl = erc20Token.tokenIconUrl ?: token.iconUrl,
+                    decimals = token.decimals,
+                    coingeckoId = erc20Token.coingeckoId,
+                    isWrapped = false,
+                    serumV3Usdc = null,
+                    serumV3Usdt = null
+                )
```
Comment (1sakov) [inline app/src/main/java/org/p2p/wallet/receive/tokenselect/ReceiveTokensPresenter.kt:78]: can we do this mapping inside the interactor?
Do we have `EthereumInteractor`?

**Review by 1sakov — APPROVED** [review]

File: app/src/main/java/org/p2p/wallet/receive/tokenselect/ReceiveTokensPresenter.kt (hunk 2 above)
Comment (kabramovp2p) [reply app/src/main/java/org/p2p/wallet/receive/tokenselect/ReceiveTokensPresenter.kt:78]: it is in Ethereum package and here we need TokenData that we can not access from EthereumInteractor

Comment (1sakov) [reply app/src/main/java/org/p2p/wallet/receive/tokenselect/ReceiveTokensPresenter.kt:78]: looks like we need to move `TokenData` to core as well. 
But it's optional

**Review by NikolaiEmelianov — APPROVED** [review]

**Pull request merged**